src/cmd.ts

```diff
@@ -374,7 +450,10 @@ async function cmdBuild(argv: BuildCmdArgvType) {
 
   const promises = [copyStatic(), ...buildPages()];
   if (appContext.base_url) {
```

Comment:
Seems like it would be be better to rely on an explicit flag / config to determine whether to generate RSS feed and podcast feed. That being said, I think it's fine for now

Reply:
Yeah, I didn't quite know the best way to do that. Having a field in the config acting as the flag felt the same as overloading an actually useful field for the same purpose. Would be interested to learn your suggested approach here and I can adjust in the future.

Reply:
Is `base_url` only used for RSS feeds? I was assuming it was used for other things too.